source/slang/slang-ir.cpp

```diff
@@ -2387,6 +2390,29 @@ namespace Slang
         return static_cast<IRStringLit*>(_findOrEmitConstant(keyInst));
     }
 
+    IRBlobLit* IRBuilder::getBlobValue(ISlangBlob* blob)
+    {
+        IRConstant keyInst;
+        memset(&keyInst, 0, sizeof(keyInst));
+
+        UnownedStringSlice inSlice((const char*)(blob->getBufferPointer()), blob->getBufferSize());
```

Comment:
We should allocate the memory from IRModule's memory arena and copy the contents over.

Reply:
Done